edb/lang/graphql/types.py

```diff
@@ -468,8 +474,10 @@ def __init__(self, schema, **kwargs):
         if edb_base is None and self.edb_type:
             edb_base = schema.edb_schema.get(self.edb_type)
 
-        assert edb_base is not None
-        assert schema is not None
+        if edb_base is None:
+            raise Exception("cannot determine the EdgeDB base type")
```

Comment:
Can you add a new exception class `GraphQLTranslationError` and use it in places like this?

Reply:
Good point, I'll add `GraphQLTranslationError` for this stuff.